### PR TITLE
dlgCountinFactor

### PR DESCRIPTION
**Pull request by lilyclements**
This dialog was disabled as it has some issues. I have updated the code to the new one and it runs the same and gives the same R code as the dialog before, however, it does come up with the same error message.

@africanmathsinitiative/developers this is ready to review

### CODE REVIEW

File: instat/dlgCountinFactor.vb [hunk 1]
```diff
@@ -1,74 +1,96 @@
...
+        clsDefaultFunction.SetRCommand("fac.nested")
+        clsDefaultFunction.SetAssignTo(strTemp:=ucrNewColName.GetText(), strTempDataframe:=ucrCountSelector.ucrAvailableDataFrames.cboAvailableDataFrames.Text, strTempColumn:=ucrNewColName.GetText())
+
+        ucrBase.clsRsyntax.SetBaseRFunction(clsDefaultFunction.Clone())
+        SetRCode(Me, ucrBase.clsRsyntax.clsBaseFunction, True)
     End Sub
+
     Private Sub ReopenDialog()
 
+    End Sub
```
Comment (maxwellfundi) [inline instat/dlgCountinFactor.vb]: This sub could be removed as it runs nothing at all.

Comment (lilyclements) [inline instat/dlgCountinFactor.vb]: Often I keep 'Reopen' subs in intentionally. They are called in the _load, and they might one day be used for something.
I'm happy to remove it when I get into the office but in a lot of my dialogs these are intentionally kept in.

**Comment by lilyclements** [comment]
@maxwellfundi This is ready for a re-review

**Comment by maxwellfundi** [comment]
@dannyparsons  this is now ready. Please have a look.

**Pull request merged**